consume.go

```diff
@@ -146,9 +158,17 @@ func WithConsumeOptionsQuorum(options *ConsumeOptions) {
 }
 
 // WithConsumeOptionsBindingExchange returns a function that sets the exchange the queue will be bound to
-func WithConsumeOptionsBindingExchange(exchange string) func(*ConsumeOptions) {
+func WithConsumeOptionsBindingExchange(name, kind string, durable, autoDelete, internal, noWait bool, args Table) func(*ConsumeOptions) {
```

Comment:
To stay more consistent with the API, can we have a separate function for each configuration,, for example `WithConsumeOptionsBindingExchangeName`, `WithConsumeOptionsBindingExchangeKind`, etc?